### DISCUSSION TITLE
LEDPOWERx with BLINK?

### DISCUSSION THREAD

**Discussion by Pewibe** [arendst/Tasmota#11580] [Ideas]
Is your feature request related to a problem? Please describe.
On devices / Tasmota rules the processing of inputs could lead to an error situation which I show to the outside world by using the (blue Wifi) LED. Because a static light do not easy catch the users attention, I show the LED flashing by use of rule timer 1 sec period. The disadvantage of this solution is the large amount of console commands, letting the initiating error situation slip away on console screen in seconds.

Describe the solution you'd like
The POWERx command supports 3 / BLINK and 4 / BLINKOFF for relays and associated leds. This should be possible for LEDPOWERx, too. The flooding of console screen with several lines per second is stopped.

**Comment by sfromis** [comment]
If you reconfigure the led pin as a Relay, you can control it using the regular Power commands, including blink.
